src/Analysis/Ast/Impl/Analyzer/PythonAnalyzerEntry.cs

```diff
@@ -248,20 +247,20 @@ public Task<IDocumentAnalysis> GetAnalysisAsync(CancellationToken cancellationTo
         }
 
         private HashSet<AnalysisModuleKey> FindDependencies(IPythonModule module, PythonAst ast, int bufferVersion) {
+            var dependencies = new HashSet<AnalysisModuleKey>();
             if (_bufferVersion > bufferVersion) {
-                return new HashSet<AnalysisModuleKey>();
+                return dependencies;
+            }
+
+            var moduleDeps = (module as IDependencyProvider)?.GetDependencies(ast).ToArray();
```

Comment:
`dependencies = module.GetDependencies(ast)` should be enough, cause `dependencies` is an empty hashset at this point

Reply:
OK